samples/beta/list-datasets.js

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(projectId = 'YOUR_PROJECT_ID', location = 'us-central1') {
+  // [START automl_video_classification_list_datasets_beta]
```

Comment:
For my own understanding - are these features available in the GA version of the API? Or only in the beta?

Reply:
The video flavor of AutoML is in beta along with Tables

But these other versions are in GA:
- Language: Entity Extraction, Sentiment Analysis, Classification
- Vision: Object Detection, Classification
- Translate